presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -169,7 +170,7 @@ public QueryStats(
             @JsonProperty("outputDataSize") DataSize outputDataSize,
             @JsonProperty("outputPositions") long outputPositions,
 
-            @JsonProperty("operatorSummaries") List<OperatorStats> operatorSummaries)
+            @JsonProperty("operatorSummaries") @Nullable List<OperatorStats> operatorSummaries)
```

Comment:
Looking at how `StageInfo` is pruned, it looks like we generally prune these list into empty lists instead of null.

In addition, all changes to this class can be reverted if the change is made that way.